enclave/src/main/java/tech/pegasys/pantheon/enclave/Enclave.java

```diff
@@ -104,7 +104,7 @@ private Request buildPostRequest(
       response = client.newCall(request).execute();
       responseBody = response.body().string();
     } catch (final IOException e) {
-      throw new EnclaveException("Failed to contact Enclave", e);
+      throw new RuntimeException("Failed to contact Enclave", e);
```

Comment:
Not sure about this change. `EnclaveException` extends `IllegalArgumentException` which is a little misleading as this is a connection error.

If this is not ok let me know.

Reply:
👍 Worse things have/will happen.